tensorrt_llm/_torch/auto_deploy/custom_ops/torch_causal_conv.py

```
@@ -0,0 +1,46 @@
+        bias,
+        stride,
+        padding,
+        dilation,
+        groups,
+    )[..., :seq_len].transpose(1, 2)
+
+
+@_torch_causal_conv1d.register_fake
+def _torch_causal_conv1d_meta(
+    input: torch.Tensor,
+    weight: torch.Tensor,
```

Ruff (0.13.1)

Unused function argument: `weight`

(ARG001)